src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -2,28 +2,30 @@
 
 from core_data_modules.logging import Logger
 from core_data_modules.util import TimeUtils
+from core_data_modules.traced_data import TracedData, Metadata
 
 from src.engagement_db_to_analysis.cache import AnalysisCache
+from src.engagement_db_to_analysis import Filters
 
 log = Logger(__name__)
 
 
 def serialise_message(msg):
-
     msg = msg.to_dict()
-    msg["timestamp"] = TimeUtils.datetime_to_utc_iso_string(msg["timestamp"])
-    msg["last_updated"] = TimeUtils.datetime_to_utc_iso_string(msg["last_updated"])
+    msg["timestamp"] = msg["timestamp"].isoformat()
```

Comment:
I think we should Message.add to_json and .from_json to pipeline-infrastructure, maybe add a todo here if you agree

Reply:
Yes now that it's fully tested in two areas we should move it. Added a Todo.